bridge/bridge/api/static/openapi/openapi.yaml

```diff
@@ -0,0 +1,919 @@
+          type: string
+          example: "1000000000000000000"
+
+    PrepareResponse:
+      type: object
+      description: Estimated outcome of a wrap or unwrap operation, including fees and net amount.
+      required:
+        - grossAmount
+        - transactionFee
+        - conversionFee
+        - totalFee
+        - netAmount
+        - diagnostics
+      properties:
+        grossAmount:
+          description: Gross value of tokens processed before deducting any fees (string-encoded integer).
+          type: string
+          example: "1000000000000000000"
+        transactionFee:
+          description: Estimated network transaction fee required for payout (string-encoded integer).
+          type: string
+          example: "2100000000000000"
+        conversionFee:
+          description: Conversion fee retained by the bridge (string-encoded integer).
+          type: string
+          example: "1000000"
+        totalFee:
+          description: Combined total of all fees applied (string-encoded integer).
+          type: string
+          example: "2105000000000000"
+        netAmount:
+          description: Net amount of tokens expected to be received after deductions (string-encoded integer).
+          type: string
+          example: "997895000000000000"
+        diagnostics:
+          description: Diagnostic information used for the conversion rate evaluation.
+          type: object
+          required:
+            - height
+            - nativeBalance
+            - wrappedBalance
+            - unwrappedBalance
+          properties:
+            height:
+              description: Block height at which diagnostic data was collected (string-encoded integer).
+              type: string
+              example: "12345678"
+            nativeBalance:
+              description: The current balance of the bridge account on the native network (string-encoded integer).
+              type: string
+              example: "200000000000000000000"
+            wrappedBalance:
+              description: The total amount of wrapped tokens that the bridge has produced since it started operating (string-encoded integer).
+              type: string
+              example: "150000000000000000000"
+            unwrappedBalance:
+              description: The total amount of wrapped tokens that the bridge has converted back into native tokens (string-encoded integer).
+              type: string
+              example: "50000000000000000000"
+
+    ErrorResponse:
+      type: object
+      description: Error response describing why an API request failed.
+      required:
+        - error
+        - errorCode
+      properties:
+        error:
+          type: string
+          description: Human-readable explanation of the error.
+          example: "Invalid address format"
+        errorCode:
+          type: string
+          description: Categorized error code when applicable.
+          nullable: true
+          enum:
```

Comment:
Are these really the only possible values?
I am missing things like "Invalid request parameters", for example.

Reply:
Yes, that is the complete list of error codes. For invalid request parameters, errorCode is set to null. I’ve added this to the list of planned API improvements, which I will address later.